### PR TITLE
Added qb-target

### PR DESCRIPTION
**Pull request by yamieboi**
Added UseTarget Optime (Tested)

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? yes
- Does your code fit the style guidelines? yes
- Does your PR fit the contribution guidelines? yes

### CODE REVIEW

**Comment by IdrisDose** [comment]
relates to #33

**Comment by milotiro95** [comment]
I'm having no such exports AddTargetModel error with this code bro normal ?

**Comment by yamieboi** [comment]
> I'm having no such exports AddTargetModel error with this code bro normal ?

no bro, i tested this before PRing...

File: config.lua [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Config = {}
 
 Config.DailyLimit = 5000
-
+Config.UseTarget = false
```
Comment (Scaarus) [inline config.lua:4]: Maybe add a comment here mentioning that qb-target is required? Having a link to the qb-target repo in the readme would be nice as well.

**Comment by KingRyuShin** [comment]
or you can just add this in qb-target : targetmodel

`    ['OpenATM'] = {
        models = {
            "prop_atm_01", 
			"prop_atm_02",
			"prop_atm_03",
			"prop_fleeca_atm"
        },
        options = {
            {
                type = "server",
                event = "qb-atms:server:doUseATM",
                icon = "fas fa-hands",
                label = "Utiliser ATM",
            },
        },
        distance = 2.5
    },`

**Comment by milotiro95** [comment]
You should update fxmanifest also to work without to restart the script. That's mine if you want to.
`fx_version 'cerulean'
game 'gta5'

description 'QB-ATM'
version '1.0.0'

shared_script 'config.lua'

server_scripts {
    '@oxmysql/lib/MySQL.lua',
    'server/main.lua',
	'config.lua'
}

client_scripts {
    '@PolyZone/client.lua',
    'client/main.lua',
    'config.lua'
}

ui_page 'nui/index.html'

files {
    'nui/images/logo1.png',
    'nui/images/logo.png',
    'nui/images/visa.png',
    'nui/images/mastercard.png',
    'nui/scripting/jquery-ui.css',
    'nui/scripting/external/jquery/jquery.js',
    'nui/scripting/jquery-ui.js',
    'nui/style.css',
    'nui/index.html',
    'nui/qb-atms.js'
}

dependencies {
    'PolyZone',
    'qb-target'
}

lua54 'yes'`

**Pull request merged**